src/subreddit.rs

```diff
@@ -210,37 +210,8 @@ pub async fn subscriptions_filters(req: Request<Body>) -> Result<Response<Body>,
 	let mut sub_list = preferences.subscriptions;
 	let mut filters = preferences.filters;
 
-	// Retrieve list of posts for these subreddits to extract display names
-	let posts = json(format!("/r/{}/hot.json?raw_json=1", sub), true).await?;
```

Comment:
Maybe it would be a better idea to handle the error here instead of propagating it, but still, this allows to save an API call. What do you think?

Reply:
I'll leave it up to @spikecodes and/or @sigaloid. In my opinion, though, we should strive for a consistent user experience across bugfixes whenever possible, so I would vote for handling the error and perhaps annotating the banned subreddit with something like a ` (banned)` prefix, e.g. `nonewnormal (banned)`.